maintenance/projects/js-toolkit/packages/liferay-npm-build-tools-common/src/project/jar.ts

```diff
@@ -1,15 +1,13 @@
 /**
- * © 2017 Liferay, Inc. <https://liferay.com>
- *
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
  * SPDX-License-Identifier: LGPL-3.0-or-later
  */
 
+import {Project} from '.';
```

Comment:
This automatic reordering is hopefully benign. Importing from "." is a bit weird, so it's not a case that we planned for in our import auto-sorter rule.

This is getting treated as an "absolute" source because it doesn't start with "./" or "..", as per [the  `isRelative()` check here](https://github.com/liferay/liferay-frontend-projects/blob/d396bc98df0cb69ff29917647929c4ff5d03044b/projects/eslint-config/plugins/liferay/lib/common/imports.js#L154-L156). I'll make a separate issue to handle this special case.

Reply:
Here's the issue: https://github.com/liferay/liferay-frontend-projects/issues/129